src/components/recordedit/form-container.tsx

```diff
@@ -136,22 +205,47 @@ const FormRow = ({ columnModelIndex }: FormRowProps): JSX.Element => {
     if (!container || !container.current) return;
 
     let cachedHeight = -1;
-    const sensor = new ResizeSensor(container.current as Element, (dimension) => {
-      const newHeight = container.current?.getBoundingClientRect().height;
-      if (newHeight === undefined || newHeight === cachedHeight || !container.current) return;
-      cachedHeight = newHeight;
-      const header = document.querySelector<HTMLElement>(`.entity-key.entity-key-${columnModelIndex}`);
-      if (header) {
-        header.style.height = `${cachedHeight}px`;
+    const sensor = new ResizeSensor(
```

Comment:
Change this block of code back to the style that it was. it was easier to read that way.